Automattic/jetpack
# Sync: listen for status changes on Woo product reviews

Fixes an issue where a product review status ( approved / unapproved ) appear incorrectly on Wordpress.com

#### Changes proposed in this Pull Request:

* Makes the list of comment types filterable
* WooCommerce sync module adds 'review' to filter

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* See: p99K0U-1mi-p2

#### Testing instructions:
- Run this branch on a connected Woo commerce site
- Add a product review to one of your products, and approve it
- Check the cache site and ensure the status of the review/comment is approved
- Unapprove the review/comment
- Check the cache site and ensure the status of the review/comment is unapproved

#### Proposed changelog entry for your changes:

* Fixes an issue where a product review status ( approved / unapproved ) appear incorrectly on Wordpress.com
